.github/pull_request_template.md

```diff
@@ -1,15 +1,19 @@
+# PR Checklist
+
+- [ ] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.
+- [ ] An Issue describing the work contained in this PR has been created on both the ufs-weather-model and any relevant subcomponent repositories.
```

Comment:
I also feel maybe this is too strict to have issues in all the subcomponent repositories. E.g. anytime there is a CCPP update,  we need to update fv3 to point to the new CCPP version (also update ufs-weather-model to point to new FV3), since CCPP has an issue already, we usually list the CCPP issue in the FV3 PR (or ufs-weather-model PR), otherwise we may have many FV3/ufs-weather-model issues just for just updating subcomponent version. I think we do need at least one issue for a set of PR, and an issue in subcomponent repository if there are scientific/technical changes in the repository.